src/Registry/Registry.ts

```diff
@@ -389,14 +389,15 @@ export const Registry:{
 			const localID:string = __getLocalID( this, pointer.$id );
 
 			const resourcesMap:Map<string, RegisteredPointer> = __getResourcesMaps( this );
-			if( resourcesMap.has( localID ) ) throw new IDAlreadyInUseError( `"${ pointer.$id }" is already being used.` );
```

Comment:
Since this error is no longer being used in the class the import should be removed.